src_cam/mo_chm_diags.F90

```diff
@@ -657,104 +709,105 @@ subroutine chm_diags( lchnk, ncol, vmr, mmr, rxt_rates, invariants, depvel, depf
    cb_aerosol_type(:,:) = 0.0_r8
    mmr_aerosol_type(:,:,:) = 0.0_r8
    ! OSLO_AERO end
-    do m = 1,gas_pcnst
-
- !...FOY (counting Fluorines, not chlorines or bromines)
-       if ( m == id_cfc12 .or. m == id_hcfc22 .or. m == id_cf2clbr .or. m == id_h1202 .or. m == id_hcfc142b &
-            .or. m == id_cof2 ) then
-          wgt = 2._r8
-       elseif ( m == id_cfc113 .or. m == id_cf3br ) then
-          wgt = 3._r8
-       elseif ( m == id_cfc114 .or. m == id_h2402 ) then
-          wgt = 4._r8
-       elseif ( m == id_cfc115 ) then
-          wgt = 5._r8
-       else
-          wgt = 1._r8
-       endif
-       if ( any( foy_species == m ) ) then
-          vmr_foy(:ncol,:) = vmr_foy(:ncol,:) +  wgt * vmr(:ncol,:,m)
-       endif
-       if ( any( tfy_species == m ) ) then
-          vmr_tfy(:ncol,:) = vmr_tfy(:ncol,:) +  wgt * vmr(:ncol,:,m)
-       endif
+   do m = 1,gas_pcnst
+
+   !...FOY (counting Fluorines, not chlorines or bromines)
+      if ( m == id_cfc12 .or. m == id_hcfc22 .or. m == id_cf2clbr .or. m == id_h1202 .or. m == id_hcfc142b &
+         .or. m == id_cof2 ) then
+         wgt = 2._r8
+      elseif ( m == id_cfc113 .or. m == id_cf3br ) then
+         wgt = 3._r8
+      elseif ( m == id_cfc114 .or. m == id_h2402 ) then
+         wgt = 4._r8
+      elseif ( m == id_cfc115 ) then
+         wgt = 5._r8
+      else
+         wgt = 1._r8
+      endif
+      if ( any( foy_species == m ) ) then
+         vmr_foy(:ncol,:) = vmr_foy(:ncol,:) +  wgt * vmr(:ncol,:,m)
+      endif
+      if ( any( tfy_species == m ) ) then
+         vmr_tfy(:ncol,:) = vmr_tfy(:ncol,:) +  wgt * vmr(:ncol,:,m)
+      endif
 
 !... counting chlorine and bromines, etc... (and total H2 species)
-       if ( m == id_ch4 .or. m == id_n2o5 .or. m == id_cfc12 .or. m == id_cl2 .or. m == id_cl2o2 .or. m==id_h2o2  ) then
-          wgt = 2._r8
-       elseif (m == id_cfc114 .or. m == id_hcfc141b .or. m == id_h1202 .or. m == id_h2402 .or. m == id_ch2br2 ) then
-          wgt = 2._r8
-       elseif ( m == id_cfc11 .or. m == id_cfc113 .or. m == id_ch3ccl3 .or. m == id_chbr3 ) then
-          wgt = 3._r8
-       elseif ( m == id_ccl4 ) then
-          wgt = 4._r8
-       else
-          wgt = 1._r8
-       endif
+      if ( m == id_ch4 .or. m == id_n2o5 .or. m == id_cfc12 .or. m == id_cl2 .or. m == id_cl2o2 .or. m==id_h2o2  ) then
+         wgt = 2._r8
+      elseif (m == id_cfc114 .or. m == id_hcfc141b .or. m == id_h1202 .or. m == id_h2402 .or. m == id_ch2br2 ) then
+         wgt = 2._r8
+      elseif ( m == id_cfc11 .or. m == id_cfc113 .or. m == id_ch3ccl3 .or. m == id_chbr3 ) then
+         wgt = 3._r8
+      elseif ( m == id_ccl4 ) then
+         wgt = 4._r8
+      else
+         wgt = 1._r8
+      endif
```

Comment:
Is there a reason for the whitespace change in this region (between here and line 810)?